app-emulation/containerd/files/containerd.service

```diff
@@ -4,7 +4,7 @@ Documentation=http://github.com/docker/containerd
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc
+ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc $CONTAINERD_OPTS
```

Comment:
> What do you mean by "break them out"?

Similar to what we do with Docker: https://github.com/coreos/coreos-overlay/blob/53aa60b0f35744f856860be6794270efd0cb04da/app-emulation/docker/files/docker.service#L14